command/daemon/command.go

```diff
@@ -54,13 +60,13 @@ func New(config Config) (Command, error) {
 		Run:   newCommand.Execute,
 	}
 
-	newCommand.cobraCommand.PersistentFlags().StringSliceVar(&Flags.Config.Dirs, "config.dirs", []string{"."}, "List of config file directories.")
-	newCommand.cobraCommand.PersistentFlags().StringSliceVar(&Flags.Config.Files, "config.files", []string{"config"}, "List of the config file names. All viper supported extensions can be used.")
+	newCommand.cobraCommand.PersistentFlags().StringSlice(Flag.Config.Dirs, []string{"."}, "List of config file directories.")
```

Comment:
The flag parsing is now more easy for clients. Flags do not have to be assigned directly to some structure. Instead configuration will be set to the configured viper.